docs/docker.rst

```diff
@@ -0,0 +1,162 @@
+Running Rally with Docker
+=========================
+
+Rally is also available as a `Docker image <https://hub.docker.com/r/elastic/rally>`_.
+
+.. _docker_limitations:
+
+Limitations
+-----------
+
+The following Rally functionality isn't supported or isn't recommended when using the Docker image:
```

Comment:
I think we can scratch the part `or isn't recommended` as we *actively* prohibit this.

Reply:
Addressed in https://github.com/elastic/rally/pull/702/commits/af1a12402329af7bf27343bc33d63d3fef7abd4b